content/docs/alerting/configuration.md

```diff
@@ -65,6 +65,9 @@ global:
   [ smtp_from: <tmpl_string> ]
   # The default SMTP smarthost used for sending emails.
   [ smtp_smarthost: <string> ]
+  # The default hostname to identify to the SMTP server. If not specified,
```

Comment:
We usually do this as `| default = "localhost"`, and it might be worth making that explicit over in the am code.

Reply:
Makes sense